### ISSUE TITLE
Make EventsList work with keyboard navigation

### ISSUE THREAD

**Issue by jmatth** [jmatth/dextools#47]
The `EventsList` component is currently implemented using a virtual scroller. This allows the filter operations to run at an acceptable speed but causes some issues with keyboard navigation:

1. Navigation order does not follow the visual order of elements in the list.
2. Navigating to the end of the list does not cause new elements to render, so the entire list cannot be traversed.
